lib/particular-calendars/canada.ts

```diff
@@ -92,67 +123,81 @@ export class Canada extends CalendarDef {
       ],
     },
 
+    // src: mr_fr_2021_ed3
     frederic_janssoone_priest: {
       precedence: Precedences.OptionalMemorial_12,
       dateDef: { month: 8, date: 5 },
     },
 
+    // src: mr_fr_2021_ed3
+    labour_day: {
+      precedence: Precedences.OptionalMemorial_12,
+      dateDef: { month: 9, nthWeekInMonth: 1, dayOfWeek: 1 },
+    },
+
+    // src: mr_fr_2021_ed3
     andre_grasset_priest: {
       precedence: Precedences.OptionalMemorial_12,
       dateDef: { month: 9, date: 2 },
     },
 
+    // src: mr_fr_2021_ed3
     dina_belanger_virgin: {
       precedence: Precedences.OptionalMemorial_12,
       dateDef: { month: 9, date: 4 },
     },
 
+    // src: mr_fr_2021_ed3
     emilie_tavernier_gamelin_religious: {
       precedence: Precedences.OptionalMemorial_12,
       dateDef: { month: 9, date: 24 },
     },
 
-    john_de_brebeuf_isaac_jogues_priests_and_companions_martyrs_copatrons_of_canada: {
-      precedence: Precedences.ProperFeast_PrincipalPatronOfARegion_8c,
-      dateDef: { month: 9, date: 26 },
-      titles: { append: [PatronTitles.CopatronOfCanada] },
-      martyrology: ['john_de_brebeuf_priest', 'isaac_jogues_priest', 'companions_martyrs'],
-    },
-
-    nereus_of_terracina_and_achilleus_of_terracina_martyrs: {
-      precedence: Precedences.ProperFeast_8f,
-      dateDef: { month: 9, date: 26 },
+    // src: mr_fr_2021_ed3
+    cosmas_of_cilicia_and_damian_of_cilicia_martyrs: {
+      dateDef: { month: 9, date: 25 },
     },
 
-    pancras_of_rome_martyr: {
-      precedence: Precedences.ProperFeast_8f,
+    // src: mr_fr_2021_ed3
+    john_de_brebeuf_isaac_jogues_priests_and_companions_martyrs: {
+      customLocaleId: 'john_de_brebeuf_isaac_jogues_priests_and_companions_martyrs_copatrons_of_canada',
+      precedence: Precedences.ProperFeast_PrincipalPatronOfARegion_8c,
       dateDef: { month: 9, date: 26 },
+      titles: { append: [PatronTitles.CopatronOfCanada] },
     },
 
+    // src: mr_fr_2021_ed3
     marie_rose_durocher_virgin: {
       precedence: Precedences.OptionalMemorial_12,
       dateDef: { month: 10, date: 6 },
     },
 
+    // src: mr_fr_2021_ed3
+    thanksgiving_day: {
+      precedence: Precedences.OptionalMemorial_12,
+      dateDef: { month: 10, nthWeekInMonth: 2, dayOfWeek: 1 },
+    },
+
+    // src: mr_fr_2021_ed3
     marguerite_dyouville_religious: {
       precedence: Precedences.ProperMemorial_11b,
       dateDef: { month: 10, date: 16 },
     },
 
+    // src: mr_fr_2021_ed3
     hedwig_of_silesia_religious: {
-      precedence: Precedences.OptionalMemorial_12,
       dateDef: { month: 10, date: 20 },
     },
 
+    // src: mr_fr_2021_ed3
     margaret_mary_alacoque_virgin: {
-      precedence: Precedences.OptionalMemorial_12,
       dateDef: { month: 10, date: 20 },
     },
 
+    // src: mr_fr_2021_ed3
     dedication_of_consecrated_churches_on_october_25: {
-      dateDef: { month: 10, date: 22 },
+      dateDef: { month: 10, date: 25 },
```

Comment:
Don’t we need to specify it should be celebrated as a solemnity? :thinking:

Reply:
This celebration is already defined in the GRC. Therefore, there's no need to redefine its `precedence`. Moreover, we are also redefining the same date, so this definition is redundant and should be removed.

However, things will change when https://github.com/romcal/romcal/issues/136 will be addressed (this celebration will be defined here instead of in the GRC).

Reply:
Okay, then consider this a non-issue.